### ISSUE TITLE
Can't find qaconv_loss

### ISSUE THREAD

**Issue by xyimaging** [ShengcaiLiao/QAConv#32]
Hello,

First of all, thanks so much for your good work!

Here is a question: inside the test_matching.py, you import `from reid.loss.qaconv_loss import QAConvLoss`, however, it seems that qaconv_loss is no longer here, so I change to other loss functions. Will it influence the performance?

Thanks!

**Comment by xyimaging** [comment]
Update: the change of loss function results in the missing of some necessary parameters for visualization, so it doesn't work. Could you help uploading the QAConvloss function or any other method to visualize the local-match results? Thanks!

**Comment by ShengcaiLiao** [comment]
Sorry, since this is a separate part, it is not always updated with the main code. For the current version, you can refer to the main.py and change QAConvLoss to TripletLoss, e.g. "criterion = TripletLoss(matcher, args.margin).cuda()".

**Comment by xyimaging** [comment]
Thanks for your quick response! After changing the loss to Tripletloss, we still need to change the code in qaconv_match.py to obtain some parameters, such as self. fc. It would be a great help if you could share the original QAConvLoss code for a better understanding of the code, thanks!

**Comment by ShengcaiLiao** [comment]
Hi, if you are using a model not trained by the latest code, then you may need all the old code. You may be able to find it here: https://github.com/ShengcaiLiao/QAConv/releases/tag/v1.0-eccv

**Comment by xyimaging** [comment]
Thanks so much!!
